Documentation/commands.md

```diff
@@ -302,3 +302,58 @@ Garbage collecting container "21b1cb32-c156-4d26-82ae-eda1ab60f595"
 Garbage collecting container "5dd42e9c-7413-49a9-9113-c2a8327d08ab"
 Garbage collecting container "f07a4070-79a9-4db0-ae65-a090c9c393a3"
 ```
+
+### rkt image extract/render
+
+For debugging or inspection you may want to extract an ACI to a directory on disk. There are a few different options depending on your use case but the basic command looks like this:
+
+```
+$ rkt image extract coreos.com/etcd etcd-extracted
+$ find etcd-extracted
+etcd-extracted
+etcd-extracted/manifest
+etcd-extracted/rootfs
+etcd-extracted/rootfs/etcd
+etcd-extracted/rootfs/etcdctl
+...
+```
+
+NOTE: A matching image must be fetched before doing this operation, Rocket will not attempt to download an image first, these subcommands will incur no-network I/O.
+
+Now there are some flags that can be added to this:
+
+To get just the rootfs use:
+
+```
+$ rkt image extract --rootfs-only coreos.com/etcd etcd-extracted
+$ find etcd-extracted
+etcd-extracted
+etcd-extracted/etcd
+etcd-extracted/etcdctl
+...
+```
+
+If you want the image rendered as it would look ready-to-run inside of the Rocket stage2 then use `rkt image render`. NOTE: this will not use overlayfs or any other mechanism. This is to simplify the cleanup, it is a simple `rm -Rf`.
+
+### rkt image cat-file
+
+For debugging or inspection you may want to extract an ACI manifest to stdout or any other file.
+If you want a file other than the manifest provide it after the image argument.
```

Comment:
hrm, I made a typo in this but the idea was `rkt image cat-file coreos.com/etcd rootfs/README.md` and `rkt image cat-file coreos.com/etcd manifest`